gallery/transforms/plot_rotated_box_transforms.py

```diff
@@ -57,100 +60,135 @@
 plot([(orig_img, orig_box)], bbox_width=10)
 
 # %%
-# Rotation
-# --------
-# Rotated bounding boxes maintain their rotation with respect to the image even
-# when the image itself is rotated through the
-# :class:`~torchvision.transforms.RandomRotation` transform.
+# Transforms illustrations
+# ------------------------
+#
+# Using :class:`~torchvision.transforms.RandomRotation`:
 rotater = v2.RandomRotation(degrees=(0, 180), expand=True)
 rotated_imgs = [rotater((orig_img, orig_box)) for _ in range(4)]
 plot([(orig_img, orig_box)] + rotated_imgs, bbox_width=10)
 
 # %%
-# Padding
-# -------
-# Rotated bounding boxes also maintain their properties when the image is padded using
-# :class:`~torchvision.transforms.Pad`.
+# Using # :class:`~torchvision.transforms.Pad`:
 padded_imgs_and_boxes = [
     v2.Pad(padding=padding)(orig_img, orig_box)
     for padding in (30, 50, 100, 200)
 ]
 plot([(orig_img, orig_box)] + padded_imgs_and_boxes, bbox_width=10)
 
 # %%
-# Resizing
-# --------
-# Rotated bounding boxes are also resized along with an image in the
-# :class:`~torchvision.transforms.Resize` transform.
-#
-# Note that the bounding box looking bigger in the images with less pixels is
-# an artifact, not reality. That is merely the rasterised representation of the
-# bounding box's boundaries appearing bigger because we specify a fixed width of
-# that rasterized line. When the image is, say, only 30 pixels wide, a
-# line that is 3 pixels wide is relatively large.
+# Using :class:`~torchvision.transforms.Resize`:
 resized_imgs = [
     v2.Resize(size=size)(orig_img, orig_box)
     for size in (30, 50, 100, orig_img.size)
 ]
 plot([(orig_img, orig_box)] + resized_imgs, bbox_width=5)
 
 # %%
-# Perspective
-# -----------
-# The rotated bounding box is also transformed along with the image when the
-# perspective is transformed with :class:`~torchvision.transforms.RandomPerspective`.
-perspective_transformer = v2.RandomPerspective(distortion_scale=0.6, p=1.0)
-perspective_imgs = [perspective_transformer(orig_img, orig_box) for _ in range(4)]
-plot([(orig_img, orig_box)] + perspective_imgs, bbox_width=10)
```

Comment:
@scotts @AntoineSimoulin , as discussed offline with both of you separately, the results of perspective were a bit fishy, so I removed it from this tutorial. We should try to investigate what's going on and potentially provide a bug fix, if we think this is a bug.

I ralso emoved Elastic below, not because it was buggy, but because it didn't seem to transform the original image in a relevant way for the purpose of this tutorial.